### PR TITLE
oop-creating_objects_from_classes-hard-q2

### PR DESCRIPTION
**Pull request by te-sa**
## PR Checklist
- [ ]  Are helper methods really needed here?
    > Opt for hardcoding input/output into the input provider when possible
- [ ]  Are the assertion fail messages good?
    - [ ]  Logical (does it make sense?)
    - [ ]  Readable (can a student understand what’s wrong?)
    - [ ]  Not too feed-y (does it give away the answer?)
    - [ ]  Grammatically sound (is the grammar and punctuation in the sentence proper?)
    - [ ]  Specific (does the fail message relate to the question?)
- [ ]  Is the code formatted well?
    > Run `CTRL` + `ALT` + `L` in IntelliJ
- [ ]  EOF newlines
- [ ]  Do the comments for Parsons follow the correct convention?
- [ ]  Is the question being tested sufficiently?
    - [ ]  Are there representative cases to test the program?
    - [ ]  What about edge cases? (empty strings, negatives, infinity, off-by-one errors, etc)
        > If the question doesn’t specify what to do, mark it as a bad question
- [ ]  Are the test method names good?
    > Test method names show up in a “What went well” section on the website, so “mathTest” will become “What went well: math test”, which doesn’t make much sense. A better name in this example would be “calculatesTaxCorrectly” so it shows up as “What went well: calculates tax correctly”

### CODE REVIEW

**Review by Hedgemon4 — CHANGES_REQUESTED** [review]
See requested changes on https://github.com/canvas-gamification/canvas-gamification-junit-tests/pull/588

File: src/oop/creating_objects_from_classes/hard/q2/MainTest.java [hunk 1]
```diff
@@ -0,0 +1,71 @@
+package oop.creating_objects_from_classes.hard.q2;
```
Comment (Hedgemon4) [inline src/oop/creating_objects_from_classes/hard/q2/MainTest.java:1]: For this one, we are actually going to need to change it. This is more of an object interactions style question, but it should actually be about manipulating initializing and manipulating the object

Comment (Hedgemon4) [reply src/oop/creating_objects_from_classes/hard/q2/MainTest.java:1]: I am currently working on an example of this, and I will send it to you once I finish it

**Review by Hedgemon4 — CHANGES_REQUESTED** [review]

**Review by Hedgemon4 — CHANGES_REQUESTED** [review]
Please review the changes I suggested on the other PRs in this category and apply them here.

**Review by Hedgemon4 — CHANGES_REQUESTED** [review]
Please take a look at #725

File: src/oop/creating_objects_from_classes/hard/q2/QuestionText.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+# oop-creating_objects_from_classes-hard-q2
+
+## Question Text
```
Comment (Hedgemon4) [inline src/oop/creating_objects_from_classes/hard/q2/QuestionText.md:3]: Is there a reason for the two different question texts?

File: src/oop/creating_objects_from_classes/hard/q2/QuestionText.md [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+- size: 5.2
+- inflated: false
+
+Then, call the `inflate` method and the `throwInWater` method on the created water toy object. The result from doing 
+this should be the printing of the `String` "Splash!".
+
+Note: You do not need to place the water toy class in your submission code, the website will handle this for you. It 
+is just for you to reference.
+
+### Dog Class:
```
Comment (Hedgemon4) [inline src/oop/creating_objects_from_classes/hard/q2/QuestionText.md:26]: Also, what is the reason for this dog class?

Comment (Hedgemon4) [reply src/oop/creating_objects_from_classes/hard/q2/QuestionText.md:26]: Not sure I understand what this question is exactly.

File: src/oop/creating_objects_from_classes/hard/q2/QuestionText.md (hunk 2 above)
Comment (SamamthaH) [reply src/oop/creating_objects_from_classes/hard/q2/QuestionText.md:3]: Sorry, copy paste issue I evidently forgot to clean up

File: src/oop/creating_objects_from_classes/hard/q2/QuestionText.md (hunk 3 above)
Comment (SamamthaH) [reply src/oop/creating_objects_from_classes/hard/q2/QuestionText.md]: also copy paste remnants, hopefully it makes more sense now

**Review by Hedgemon4 — APPROVED** [review]

**Pull request merged**